src/main/c/truffleposix/truffleposix.c

```diff
@@ -33,17 +33,27 @@ OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 SUCH DAMAGE.
 */
 
+/* For clock_gettime() and lstat() on Linux */
```

Comment:
I am not sure whether we want to also work on C89 and other comments in the file use the same style, I tried C99 since it seems easier and works in CI.